### PR TITLE
[ fix ] Merging Go-Backend changes to Production .

### PR DESCRIPTION
**Pull request by bhaskarhc**
**What this PR does / why we need it**:
-  Using GO-Backend having endpoints .
       - https://openebs.ci/api/build/
       -  https://openebs.ci/api/packet/v11
       - https://openebs.ci/api/packet/v12
       - https://openebs.ci/api/packet/v13
- Pipeline NA shows :no_entry_sign: icon 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
     <div class="row">
-        <div class="col-md-2 bg-white border p-0 scrollbar" id="scrollbar">
+        <div class="col-md-2 bg-white p-0 scrollbar" id="scrollbar">
             <app-sidebar></app-sidebar>
         </div>
         <div class="col-md-10 p-0 mainBody" >
             <app-header></app-header>
-            <div class="scrollbar bodyContent" id="scrollbar">
+            <div class="fixed-bottom bg-white ">
+
```
Comment (chandankumar4) [inline src/app/app.component.html]: Remove the extra space

File: src/app/overview/overview.component.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.img-overview{
+    height: 260px;
+    /* border: solid 1px #CCC;
```
Comment (chandankumar4) [inline src/app/overview/overview.component.css]: Remove the commented code

File: src/app/overview/overview.component.html [hunk 3]
```diff
@@ -1,27 +1,55 @@
...
-    <p class="py-3" style="color:#54576b">This CI/E2E portal is built to be used both by OpenEBS developer and user community. Developers and users get
-      the
-      detailed test report across various release branches include the master branch. </p>
-    <img class="img-fluid" src="/assets/images/cidashboard.png" style="padding-right:20px;padding-left:5px" alt="">
-    <p class="py-5" style="color:#54576b">The successful builds of OpenEBS from master are deployed onto various live deployments that are hosted on
-      Kubernetes cluster deployed currently on GKE. These reference stateful workloads are deployed for both the
-      storage
-      engines Jiva and cStor. </p>
-      <img class="img-fluid" src="/assets/images/workload-dashboard.png" style="padding-right:20px;padding-left:5px" alt="">
+    <h1 class="my-3 text-center rounded shadow-sm"   >CI/E2E Portal Overview</h1>
```
Comment (chandankumar4) [inline src/app/overview/overview.component.html]: Remove the extra space after class name

File: src/app/overview/overview.component.html [hunk 4]
```diff
@@ -1,27 +1,55 @@
...
+          <p class="py-5" style="color:#54576b">The successful builds of OpenEBS from master are deployed onto various live deployments that are hosted on
+              Kubernetes cluster deployed currently on GKE. These reference stateful workloads are deployed for both the
+              storage
+              engines Jiva and cStor. </p>
+      </div>
+    </div>
+    <h3 class="my-5">CI/E2E dashboard:</h3>
+    <div class="row my-5 py-5 border-left border-right rounded">
+      <div class="col-md-3 text-center ">
+          <p class="py-3" style="color:#54576b">This CI/E2E portal is built to be used both by OpenEBS developer and user community. Developers and users get
```
Comment (chandankumar4) [inline src/app/overview/overview.component.html]: Merge the three line content to two lines

File: src/app/services/agile.services.ts [hunk 5]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (chandankumar4) [inline src/app/services/agile.services.ts]: Remove the extra space

File: src/app/services/kubernetes.service.ts [hunk 6]
```diff
@@ -1,65 +1,75 @@
 import { Injectable } from "@angular/core";
 import { HttpClient } from "@angular/common/http";
 import { Subject } from "rxjs";
+import {Observable} from "rxjs/Observable";
+import { forkJoin } from "rxjs/observable/forkJoin";
+import 'rxjs/add/operator/map'
+
```
Comment (chandankumar4) [inline src/app/services/kubernetes.service.ts]: Remove the blank line

File: src/app/sidebar/sidebar.component.html [hunk 7]
```diff
@@ -15,10 +15,14 @@
                         <h5 translate>SIDEBAR.HOME</h5>
                     </div>
                 </li>
-                <li *ngIf="vendor == 'false'" class="item cursor-pointer sidebarItems rounded-left button text-center mt-5" routerLink="/" [routerLinkActiveOptions]="{ exact : true }"
+                <li *ngIf="vendor == 'false'" class="item cursor-pointer sidebarItems rounded-left button text-center mt-5 ciCd" routerLink="/" [routerLinkActiveOptions]="{ exact : true }"
                     routerLinkActive="active" >
                     <div class="text-center">
-                        <img src="assets/images/company-logos/ci-e2e.png" alt="ci-e2e">
+                        <div class="ciCdAnime">
```
Comment (chandankumar4) [inline src/app/sidebar/sidebar.component.html]: Where this div section is used?

File: src/app/table/table.component.ts [hunk 8]
```diff
@@ -41,55 +66,56 @@ export class TableComponent implements OnInit {
...
-        this.restItemsUrl = "https://openebs.ci/api/";
-    } else {
-      this.restItemsUrl = "https://staging.openebs.ci/api/";
-    }
-    this.meta.addTag({ name: 'description', content: 'Openebs.ci is the CI and E2E portal for OpenEBS github PRs. Using this portal, OpenEBS community uses this portal to find the build quality against each PR that is merged. It also hosts stateful application workloads with various deployment scenarios. The example workloads include MongoDB, Percona, WordPress, Prometheus, Redis, CockroachDB etc.' });
-    this.meta.addTag({ name: 'keywords', content: 'Openebs,EBS,workload,mongo,jiva,cstor' });
-    this.titleService.setTitle( "CI/E2E dashboard for OpenEBS" );
   }
-
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Remove this line

File: src/app/table/table.component.ts [hunk 9]
```diff
@@ -41,55 +66,56 @@ export class TableComponent implements OnInit {
...
-    this.meta.addTag({ name: 'description', content: 'Openebs.ci is the CI and E2E portal for OpenEBS github PRs. Using this portal, OpenEBS community uses this portal to find the build quality against each PR that is merged. It also hosts stateful application workloads with various deployment scenarios. The example workloads include MongoDB, Percona, WordPress, Prometheus, Redis, CockroachDB etc.' });
-    this.meta.addTag({ name: 'keywords', content: 'Openebs,EBS,workload,mongo,jiva,cstor' });
-    this.titleService.setTitle( "CI/E2E dashboard for OpenEBS" );
   }
-
+  
   ngOnInit() {
     localStorage.removeItem('row');
-    //for Highlight the row
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Remove this line

File: src/app/table/table.component.ts [hunk 10]
```diff
@@ -206,204 +162,204 @@ export class TableComponent implements OnInit {
...
+  pipelineStatus(index, data) {
+    try {
+      if(data[index].status == "running") {
+        return "running"
+      } else if(data[index].status == "pending") {
+        return "pending"
+      } else if(data[index].status == "cancelled") {
+        return "cancelled"
+      } else if(data[index].status == "none") {
+        console.log("none data ........................................")
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Remove the console line

File: src/app/workloads/workloads.component.html [hunk 11]
```diff
@@ -154,6 +141,7 @@
                 </div>
               </div>
             </div>
+
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.html]: Remove the blank line

File: src/app/workloads/workloads.component.ts [hunk 12]
```diff
@@ -167,6 +167,7 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
     // this.routeSub = this.route.params.subscribe(params =>{
     // });
 
+
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: Remove this line

File: src/app/workloads/workloads.component.ts [hunk 13]
```diff
@@ -175,6 +176,7 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
       this.dashboardurl = res.dashboardurl;
       this.openebsengine = res.openebsEngine;
       this.titleService.setTitle(this.workloadName + " dashboard | OpenEBS.io");
+      //  this.kubernetsServices.setApiUrl(res.urlApi);
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: Remove the commented code

File: src/app/workloads/workloads.component.ts [hunk 14]
```diff
@@ -215,8 +217,8 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
           this.pvc = res.pvc;
           this.pvctemp = res.pvc;
           this.pvcarray = this.pvctemp.pvc;
-          this.workloadImage = this.statefullSets[0].dockerImage;
-          this.dockerImage = this.jivaContrllers[0].openebsjivaversion;
+          // this.workloadImage = this.statefullSets[0].dockerImage;
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: Remove the commented code from all place

Comment (inyee786) [reply src/app/workloads/workloads.component.ts]: it will be required for futrue

File: src/app/sidebar/sidebar.component.html (hunk 7 above)
Comment (bhaskarhc) [reply src/app/sidebar/sidebar.component.html]: This is logo of cicd ,

Comment (chandankumar4) [reply src/app/sidebar/sidebar.component.html]: okay

**Pull request merged**